src/data/faq.json

```diff
@@ -1879,6 +1879,67 @@
+                            "Additionally, in the risk calculation, several low-risk encounters over the course of a day can lead to the display of an encounter with increased risk. However, this may not necessarily have been the case with another device, since it did not necessarily record the same contacts in the same period.",
+                            "In addition, due to different technical equipment (hardware) and system software alone, they react differently with regard to transmission power/signal strength and reception sensitivity. Therefore, even two adjacent devices do not necessarily receive the same diagnostic keys at the same time with identical parameters.",
+                            "See FAQ: <ul><li><a href='#risk_encounter_not_realtime' target='_blank'>Why do I receive a risk warning several days after the risk encounter?</a></li><li><a href='#risk_encounter_no_warning' target='_blank'>Why don't I get a risk encounter warning even though I've had contact with someone who has a risk encounter displayed in their app?</a></li><li><a href='#pos_test_no_risk_encounter' target='_blank'>I tested positive. Why does a person in the same household not receive a risk warning?</a></li><li><a href='#risk_encounter_doubted' target='_blank'>Why do I receive a risk warning for a date when I had no contacts / did not leave the house?</a></li><li><a href='#risk_encounter_place' target='_blank'>Are people and/or places assigned to a risk encounter?</a></li></ul>"
+                        ]
+                    },
+                    {
+                        "title": "I tested positive. Why does a person in the same household not receive a risk warning?",
+                        "anchor": "pos_test_no_risk_encounter",
+                        "active": false,
+                        "textblock": [
+                            "In order to receive warnings, there are certain requirements that must be met:<ul><li>After receiving the positive test result, it must be voluntarily shared by the user via the Corona-Warn-App. If the test result could not be determined via the app, the positive PCR test result can be entered and shared via a TAN.</li><li>The app must have been installed on both devices <b>before</b> sharing the positive test result via the Corona-Warn-App, otherwise no encounters were recorded.</li><li>The two devices must have been close enough and long enough <b>before</b> sharing the test result. Encounters <b>after</b> sharing a positive result are of course <b>not</b> warned.</li><li>The time period for the onset of the first symptoms given when sharing the test result, as well as the type or intensity of the symptoms, may also play a role in the risk calculation. So it also depends on which days before sharing the test result the people's devices recorded the encounters. In some cases, this may result in no warning being sent.</li></ul>",

```

Comment:
"the people's devices" =>
the person's device"

In the title it refers to a singular person: "Why does a person in the same household not receive a risk warning?" so the above also has to be singular.

Reply:
Changed it in https://github.com/corona-warn-app/cwa-website/pull/2529/commits/e25bfbeb50abbc2054e6fed0564c3cd2e725c5c9, thanks!

Reply:
It now says "person's devices" with devices in the plural form. Most people would only be using one device, so that it why I suggested "person's device".